packages/dds/tree/src/feature-libraries/forestIndex.ts

```diff
@@ -105,78 +107,60 @@ export class ForestIndex implements Index<unknown>, SummaryElement {
     }
 
     getAttachSummary(
```

Comment:
Is it worth sharing some code between this and `summarize`?

Reply:
Yea that's a cool idea. A function which takes the two strings and returns the summary tree. Let's call it... `summarizeCore` :D